docs/components/table.md

```diff
@@ -257,6 +254,113 @@ export default {
 </script>
 ```
 
+### Custom Row Attributes
+
+There is a prop called `hasSideBorder` that when passed in, enables a default side border on each row, using the color `steal-200` by default.
+It also adds a default hover color on the row that uses the color `steal-100` by default. These colors can be overridden by themes or by using
+custom row attributes. This prop also causes the row borders to be separate and enables spacing between rows.
+Example using `hasSideBorder` prop:
+
+<template>
+  <KTable
+    :options="tableOptions"
+    hasSideBorder
+    />
+</template>
+
+```vue
+<template>
+  <KTable
+    :options="tableOptions"
+    hasSideBorder
+    />
+</template>
+```
+
+We can also add custom row attributes via a prop called `rowAttrs` - this prop is a Function that returns an object comprising the conditional custom attributes
+that you want the rows to have. `hasSideBorder` prop must be passed in to enable custom row attributes.
+In the following example, the rows have a custom green color if enabled is true, and yellow if enabled is false, coming from the
+custom `rowAttrs` function. We can also add a custom data-testid onto each row using these attributes (you can inspect it to see there is a custom data-testid attribute with `rowItem` on each row).
+
+<template>
+  <KTable
+    :options="tableOptions"
+    hasSideBorder
+    :rowAttrs="rowAttrsFn"
+    />
+</template>
+
+```vue
+<template>
+  <KTable
+    :options="tableOptions"
+    :rowAttrs="rowAttrsFn"
+    hasSideBorder
+    />
+</template>
+<script>
+import { defaultSorter } from '@kongponents/KTable'
+export default {
+  data() {
+    return {
+      row: null,
+      eventType: '',
+      tableOptions: {
+        headers: [
+          { label: 'Type', key: 'type' },
+          { label: 'Value', key: 'value' }
+        ],
+        data: [
+          {
+            name: 'Basic Auth',
+            id: '517526354743085',
+            enabled: 'true',
+            themeColors: []
```

Comment:
lets remove this key, to me its confusing when in an example about theming and its not used to theme